amqp/src/main/scala/akka/stream/alpakka/amqp/scaladsl/AmqpFlow.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.amqp.scaladsl
+
+import akka.NotUsed
+import akka.stream.alpakka.amqp.{AmqpWriteSettings, WriteMessage}
+import akka.stream.alpakka.amqp.impl.AmqpPublishFlowStage
+import akka.stream.scaladsl.{Flow, Keep}
+import akka.util.ByteString
+
+object AmqpFlow {
+
+  /**
+   * Scala API: Creates an [[AmqpFlow]] that accepts (ByteString, passthrough) elements.
+   *
+   * This stage materializes to a `Future[Done]`, which can be used to know when the Flow completes, either normally
+   * or because of an amqp failure
+   */
+  def simple[O](settings: AmqpWriteSettings): Flow[(ByteString, O), O, NotUsed] =
```

Comment:
I'm not sure if `simple` adds so much value.

Reply:
Not sure either, here I simply followed what was done in other dsl (RPC & Sink for example).
I can remove it if you prefer.